plugins/asdf/asdf.plugin.zsh

```diff
@@ -4,7 +4,7 @@ ASDF_COMPLETIONS="$ASDF_DIR/completions"
 
 # If not found, check for Homebrew package
 if [[ ! -f "$ASDF_DIR/asdf.sh" || ! -f "$ASDF_COMPLETIONS/asdf.bash" ]] && (( $+commands[brew] )); then
-   ASDF_DIR="$(brew --prefix asdf)"
+   ASDF_DIR="$(brew --prefix asdf)/libexec"
    ASDF_COMPLETIONS="$ASDF_DIR/etc/bash_completion.d"
```

Comment:
ASDF_COMPLETIONS does not evaluate to a valid path with this change. /etc/bash_completions.d lives under the root asdf directory rather than under libexec.

Reply:
```suggestion
   ASDF_COMPLETIONS="$(brew --prefix asdf)/etc/bash_completion.d"
```

Reply:
Applied changes, @orien thanks!